frontend/assets/style/_utilities.scss

```diff
@@ -0,0 +1,52 @@
+.is-link-inherit {
+  color: inherit;
+  cursor: pointer;
+  text-decoration: underline;
+
+  &:hover,
+  &:focus {
+    color: $link;
+  }
+}
+
+.is-unstyled {
+  background: none;
+  border: none;
+  font-size: inherit;
+  color: inherit;
+  text-align: inherit;
+  padding: 0;
+
+  &:focus,
+  &:hover {
+    outline: none;
+    border: none;
+    color: inherit;
+  }
+}
+
+.sr-only {
```

Comment:
Can you put a comment above this to explain what `.sr-only` means?

Reply:
Screen reader only? Sure